frames:
  map: "map"
  odom: "odom"
  base_link: "xiroi"
  sensors:
    imu: "imu"
    gps: "gps"
    buoy: "gps"
    origin_suffix: "origin"
  robot: "xiroi"